src/frontend/components/DashboardVideoLiveControlPane/tab/DashboardVideoLiveTabAttendance/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { POLL_FOR_ATTENDANCES } from 'default/sideEffects';
+import { liveState, Video } from 'types/tracks';
+
+const messages = defineMessages({
+  loading: {
+    defaultMessage: 'Loading attendances...',
+    description:
+      'Accessible message for the spinner while loading the attendances.',
+    id: 'components.DashboardVideoLiveControlPane.tab.DashboardVideoLivettendance.loading',
+  },
+  noParticipants: {
+    defaultMessage: 'The live has no participant yet',
+    description: 'Message displayed if the live has no participant yet',
+    id: 'components.DashboardVideoLiveControlPane.tab.DashboardVideoLivettendance.noParticipants',
+  },
+});
+
+interface DashboardVideoLiveTabAttendanceProps {
+  video: Video;
+}
+
+export const DashboardVideoLiveTabAttendance = ({
+  video,
+}: DashboardVideoLiveTabAttendanceProps) => {
+  const intl = useIntl();
+  const interval =
+    video.live_state &&
+    [liveState.STOPPING, liveState.RUNNING].includes(video.live_state)
+      ? POLL_FOR_ATTENDANCES
+      : false;
+
+  const noRefresh = interval === false;
+  // only if the video is running or is stopping, we refresh the list
+  const { data, status } = useLiveAttendances({
+    refetchInterval: interval,
+    refetchIntervalInBackground: !noRefresh,
+    refetchOnWindowFocus: !noRefresh,
+  });
+
+  switch (status) {
+    case 'idle':
+    case 'loading':
+      return (
+        <Box width="full">
+          <Spinner size="large">
+            <FormattedMessage {...messages.loading} />
+          </Spinner>
+        </Box>
+      );
+    case 'error':
+      return (
+        <Box width="full">
+          <ErrorMessage code="generic" />
+        </Box>
+      );
+
+    case 'success':
+      return (
+        <Box width="full">
+          {data!.count > 0 && (
+            <Box
+              background="white"
+              direction="column"
+              margin={{ top: 'small' }}
+              pad={{ top: 'medium' }}
+              round="6px"
+              style={{
+                boxShadow: '0px 0px 6px 0px rgba(2, 117, 180, 0.3)',
+                minHeight: '70px',
+              }}
+            >
+              {data?.results.map((liveSession) => (
+                <DashboardVideoSessionLiveTabAttendance
+                  key={liveSession.id}
+                  liveSession={liveSession}
+                />
+              ))}
+            </Box>
+          )}
+          {data?.count === 0 && (
+            <Box
+              background={{
+                image: `url(${appData.static.img.liveBackground})`,
+                position: 'top',
+                repeat: 'no-repeat',
+                size: 'cover',
+              }}
+              flex="grow"
+              margin={{ top: 'small' }}
+              pad={{ top: 'small' }}
+            >
+              <Box
+                margin="auto"
+                pad={{ horizontal: 'none', vertical: 'large' }}
+              >
```

Comment:
This box is probably useless

Reply:
this one is needed to do the expected rendering